### ISSUE TITLE
DummySIM isn't available

### ISSUE THREAD

**Issue by thomasmfish** [MicronOxford/cockpit#850]
For some reason the dummy SIM that is in Microscope isn't in Cockpit. I've made the change in a branch on my fork (see https://github.com/MicronOxford/cockpit/commit/fa509b4003cf9523b40dd01d8ee0945e89ad32b3), though I was only after it for GUI testing purposes

**Comment by iandobbie** [comment]
You are correct we never implemented dummy SIM in cockpit as basically it requires the hardware triggers to synchronize all the hardware and it just seemed too much effort for little benefit.

**Comment by thomasmfish** [comment]
Ah okay, I was only after another GUI elements to test how resizing & minimum sizing worked with wrapping the elements

**Comment by thomasmfish** [comment]
Feel free to close this then!

**Comment by iandobbie** [comment]
Seems pretty trivial though so introducing it is pretty easy. 

There is now also a digital IO device in microscope and cockpit so we could easily add a dummy DIO device
